lib/config/react.js

```diff
@@ -1,8 +1,6 @@
 const merge = require('merge');
 
 module.exports = {
-  extends: 'plugin:shopify/esnext',
```

Comment:
I don't think `react` should be a base config. I had a lot of trouble having a non-react typescript project without configs being unintentionally overwritten. This is a breaking change.

Reply:
So the new world is typescript is a base config, react is an augment? I like it!

Seems the [Provided configs](https://github.com/Shopify/eslint-plugin-shopify#provided-configurations) part of the readme needs some love - to add mention of the typescript core config, moving react to an augment, and removing typescript-prettier as that no longer exists